### PR TITLE
[KYUUBI #5638][AUTHZ] URI related privileges shall be READ/WRITE

### LINKED ISSUE #5638: [Subtask] URI related privileges shall be READ/WRITE
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/kyuubi/issues) and found no similar issues.

### Describe the subtask

URI related privileges shall be READ/WRITE

### Parent issue

#5368

### Are you willing to submit PR?

- [X] Yes. I would be willing to submit a PR with guidance from the Kyuubi community to improve.
- [ ] No. I cannot submit a PR at this time.

### PR DESCRIPTION
**Pull request by yaooqinn**
### _Why are the changes needed?_

This PR fixes #5638 which targets URI related privileges to select and update

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

### _Was this patch authored or co-authored using generative AI tooling?_

no

### CODE REVIEW

File: extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/AccessType.scala [hunk 1]
```diff
@@ -28,16 +28,18 @@ object AccessType extends Enumeration {
   val NONE, CREATE, ALTER, DROP, SELECT, UPDATE, USE, READ, WRITE, ALL, ADMIN, INDEX = Value
 
   def apply(obj: PrivilegeObject, opType: OperationType, isInput: Boolean): AccessType = {
+    if (obj.privilegeObjectType == DFS_URI || obj.privilegeObjectType == LOCAL_URI) {
+      // This is equivalent to ObjectType.URI
+      return if (isInput) READ else WRITE
+    }
```
Comment (AngersZhuuuu) [inline extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/AccessType.scala:34]: Nice catch!

**Review by AngersZhuuuu — APPROVED** [review]

**Comment by yaooqinn** [comment]
cc @bowenliang123

**Review by bowenliang123 — APPROVED** [review]
LGTM

**Comment by pan3793** [comment]
Thanks, merged to master

**Pull request closed without merging**